tests/test_cli.py

```diff
@@ -63,3 +65,155 @@ def test_config_compatibility(self):
                 execute_subprocess_async(
                     self.base_cmd + ["--config_file", str(config), self.test_file_path], env=os.environ.copy()
                 )
+
+
+class PodConfigTester(unittest.TestCase):
+    """
+    Test case for verifying the `accelerate pod-config` CLI passes the right `gcloud` command.
+    """
+
+    _tpu_name = "test-tpu"
+    _tpu_zone = "us-central1-a"
+    _command = "ls"
+    cmd = ["accelerate", "pod-config"]
+    _base_output = "cd /usr/share"
+    _command_file = "tests/test_samples/test_command_file.sh"
```

Comment:
That's a test class which is inherently private, thus we don't need to add all those `_` that hurt readability :-)